docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml

```diff
@@ -28,17 +30,25 @@
   role="bold">]</emphasis> ,<emphasis role="bold"> [</emphasis>
   <emphasis>allowoverwrite</emphasis> <emphasis role="bold">] , [</emphasis>
   <emphasis>replicate</emphasis> <emphasis role="bold">] , [</emphasis>
-  <emphasis> compress </emphasis> <emphasis role="bold">] , [</emphasis>
-  <emphasis> sourceCsvEscape </emphasis> <emphasis role="bold">])</emphasis>
-  <emphasis></emphasis></para>
+  <emphasis> compress </emphasis><emphasis role="bold">]</emphasis> ,
+  <emphasis role="bold"> [</emphasis> <emphasis> sourceCsvEscape </emphasis>
+  <emphasis role="bold">]</emphasis>, <emphasis role="bold"> [</emphasis>
+  <emphasis> failIfNoSourceFile </emphasis> <emphasis
+  role="bold">]</emphasis>, <emphasis role="bold"> [</emphasis> <emphasis>
+  recordStructurePresent </emphasis> <emphasis role="bold">]</emphasis>,
+  <emphasis role="bold"> [</emphasis> <emphasis> quotedTerminator </emphasis>
+  <emphasis role="bold">]</emphasis>, <emphasis role="bold">[</emphasis>
+  <emphasis>expireDays</emphasis> <emphasis role="bold">])</emphasis></para>
```

Comment:
One parameter missing, the 'encoding' between 'quotedTerminator' and 'expireDays'.
'... [quotedTerminator ], [encoding ], [expireDays])'